apps/core-app/src/main/modules/database/index.ts

```
@@ -239,29 +242,117 @@ export class DatabaseModule extends BaseModule {
     }
   }
 
+  private getDbSchedulerBusyReason():
+    | {
+        reason: 'db-write-scheduler'
+        queued: number
+        processing: boolean
+        currentTaskLabel?: string
+      }
+    | null {
```

ESLint

[error] Expected indentation of 6 spaces

(style/indent-binary-ops)